examples/helpers/android/app/CMakeLists.txt.in

```diff
@@ -15,7 +15,7 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.9)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.5)
```
I wouldn't change the CMake version for Android examples, because Gradle uses 3.6 by default. And examples' `CMakeLists` are not part of the OLP SDK's main CMake setup process - they run independently by Gradle.

Reply:
Agree